getting-started/iterate-new-app-intellij.hbs.md

```diff
@@ -0,0 +1,141 @@
+   1. In the **Source Image** field, provide the destination image repository to publish an image containing your workload source code. The Source Image value tells the Tanzu Developer Tools extension where to publish the container image with your uncompiled source code, and what to name that image. The image must be published to a container registry where you have write (push) access.
+   For example, `gcr.io/myteam/tanzu-java-web-app-source`.
+
+   >**Note** Consult the documentation for the registry you're using to determine which steps are necessary to authenticate and gain push access.
+
+   >For example, if you use docker consult [docker's docs](https://docs.docker.com/engine/reference/commandline/login/), if you use Harbor consult [Harbor's docs](https://goharbor.io/docs/1.10/working-with-projects/working-with-images/pulling-pushing-images/), etc.
+
+1. In the Project tab of IntelliJ, right-click the `Tiltfile` file under the application name `tanzu-java-web-app` and select `Run \'Tanzu Live Update - tanzu-java-web-app\'` to begin Live Updating the application on the cluster.
+1. Alternatively, select the `Edit Run/Debug configurations` dropdown in the top-right corner, select `Tanzu Live Update - tanzu-java-web-app`, then click the green play button to the right of the `Edit Run/Debug configurations` dropdown.
+The `Run` tab will open and display the output from Tanzu Application Platform and from Tilt indicating that the container is being built and deployed.
+    - On the `Tanzu Panel` tab you will see the status of Live Update reflected under the `tanzu-java-web-app` workload entry.
+    - Live update can take 1 to 3 minutes while the workload deploys and the Knative service becomes available.
+
+    >**Note** Depending on the type of cluster you use, you might see an error similar to the following:
+
+    >`ERROR: Stop! cluster-name might be production.
+    >If you're sure you want to deploy there, add:
+    >allow_k8s_contexts('cluster-name')
+    >to your Tiltfile. Otherwise, switch k8scontexts and restart Tilt.`
+    >Follow the instructions and add the line, `allow_k8s_contexts('cluster-name')` to your `Tiltfile`.
+
+2. When the Live Update task in the `Run` tab says that it was successful, resolve to "Live Update Started," use the hyperlink at the top of the Run output following the words **Tilt started on** to  view your application in your browser.
+3. In the IDE, make a change to the source code. For example, in `HelloController.java`, edit the string returned to say `Hello!` and save.
+4. The container is updated when the logs stop streaming. Navigate to your browser and refresh the page.
+5. View the changes to your workload running on the cluster.
+6. Either continue making changes, or stop and deactivate the live update when finished. To stop Live Update navigate to the `Run` tab at the bottom left of the IntelliJ window and click the red stop icon on the left side of the screen.
```

Comment:
and deactivate?

Reply:
great catch, no idea where this came from, it must be a relic from MVP times